baselines/asset/src/v1/asset_service_client.ts.baseline

```diff
@@ -84,6 +84,13 @@ export class AssetServiceClient {
     }
     opts.servicePath = opts.servicePath || servicePath;
     opts.port = opts.port || port;
+
+    // users can override the config from client side, like bundling options.
+    // The detailed structure of the clientConfig can be found here: https://github.com/googleapis/gax-nodejs/blob/master/src/gax.ts#L546
+    // The way to override bundling options:
+    //
+    // const customConfig = {"interfaces": {"service": {"methods": {"methodName": {"bundling": {..}}}}}}
+    // const client = new Client({ projectId, customConfig });
```

Comment:
we can use the real client name here instead of `Client`. Also, let's not talk about `bundling` because it's a very special use case (only for logging and pubsub). Let's just say that the config can be overridden.

Reply:
We can even use a real service name and a real method name (e.g. the first one).

Reply:
Ok, so we wanna give example of other config with the real name, maybe `showcase`